R/convert-from-datacamp.R

```diff
@@ -25,82 +25,65 @@ datacamp_to_learnr_pkg <-
 
     # options(usethis.quiet = TRUE)
 
-    if (is.null(cur_path)) {
-      cur_path = getwd()
+    if (is.null(dc_path)) {
+      dc_path = getwd()
```

Comment:
this isn't necessary, since the default is already `"."`. There would not be null unless the user explicitly entered `NULL`.